multixstar_logging.py

```diff
@@ -160,12 +145,12 @@ def get_xcmds(args=[], binpath=""):
     binpath += "/"
     if len(args) > 0:
         if not os.path.exists("../" + args[0]):
-            to_return = 1 + "\n"
```

Comment:
i don't think `to_return` is used (I can't see it anywhere). 
It must be a relic of an older version of the code.